### PR TITLE
EEJT-0001

### PR DESCRIPTION
**Pull request by treep25**

### CODE REVIEW

File: multi-project/core/src/main/java/com/epam/Utils.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+public class Utils {
+    public static boolean isAllPositiveNumbers(String... str) {
+        if (str.length > 0) {
+            boolean flag = false;
+            for (String number : str) {
+                flag = StringUtils.isPositiveNumber(number);
+            }
+            return flag;
+        }
+        return false;
```
Comment (OlehHoliuk) [inline multi-project/core/src/main/java/com/epam/Utils.java:14]: Rewrite this using Stream API. Use `allMatch` method

Comment (OlehHoliuk) [inline multi-project/core/src/main/java/com/epam/Utils.java:14]: Cover with unit tests, please.

File: multi-project/core/src/main/java/com/epam/Utils.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.epam;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class Utils {
+    public static boolean isAllPositiveNumbers(String... str) {
+        if (ArrayUtils.isNotEmpty(str) && str.length > 0) {
```
Comment (OlehHoliuk) [inline multi-project/core/src/main/java/com/epam/Utils.java:10]: `str.length > 0` is redundant here. `ArrayUtils::sNotEmpty` already has this check inside

**Review by OlehHoliuk — CHANGES_REQUESTED** [review]

File: multi-project/core/src/main/java/com/epam/Utils.java (hunk 2 above)
Comment (treep25) [reply multi-project/core/src/main/java/com/epam/Utils.java]: done , remove redundant str.length > 0

File: multi-project/core/src/test/java/com/epam/UtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        String[] arrayTestObjs = new String[]{"     ", "   ", ""};
+
+        assertFalse(Utils.isAllPositiveNumbers(arrayTestObjs));
+    }
+
+    @Test
+    void returnFalseWhenNullCaseTest() {
+        String[] arrayTestObjs = new String[]{};
+
+        assertFalse(Utils.isAllPositiveNumbers(arrayTestObjs));
```
[the remark this answers is not in the document]
Comment (treep25) [reply multi-project/core/src/test/java/com/epam/UtilsTest.java]: done, added nullCaseTest

File: multi-project/core/src/main/java/com/epam/Utils.java (hunk 1 above)
Comment (treep25) [reply multi-project/core/src/main/java/com/epam/Utils.java]: done

**Pull request merged**